examples/demo/src/index.ts

```diff
@@ -39,6 +75,16 @@ languages.css.cssDefaults.setOptions({
   },
 });
 
+languages.json.jsonDefaults.setDiagnosticsOptions({
+  allowComments: true,
```

Comment:
:D

Reply:
over kill json ^^

Reply:
Since the Monaco already supports jsonc by default I figured: _why not embrace it?_

I think quickly toggling comments is a powerful feature, especially when fiddling, which is what this demo is for. :)

Reply:
yeah sure and also allowing trailing commas does help ^^ -> now also allow any type of quote or no quotes at all -> or even some spreads and objects -> or functions -> ooops javascript xD

Reply:
Actually [json5](https://github.com/json5/json5) is really cool! Unfortunately Monaco doesn’t support it by default.